packages/deployer/.gitignore

```diff
@@ -11,7 +11,6 @@ coverage/
 *config.json
 *secrets.env
 *secrets.json
-*receipt.json
```

Comment:
I found it strange that this is listed in `.gitignore`, since you are not required to run deployments from `deployer` directory...
So, we could add it to the root folder - but I'd just not include it anywhere...

Reply:
It's more of a convenience for a developer. If I run the deployer with the defaults it creates the `receipt.json` in the `output` directory and I may add it by mistake when committing changes if it's not in the `.gitignore`.

Reply:
Ok, I'll add it back.